package.json

```diff
@@ -199,6 +200,7 @@
     "npm": "^10.7.0"
   },
   "overrides": {
-    "colors": "1.4.0"
-  }
+    "colors": "1.4.0",
+    "vue-material-design-icons" : "$vue-material-design-icons"
```

Comment:
Isn't this no-op when you've already changed the dependency to `file:...` above?

Reply:
no